### ISSUE TITLE
no smart input popup for trees on built maqetta

### ISSUE THREAD

**Issue by childsb** [maqetta/maqetta#2763]
(following the test plan for trees)
1. Choose New HTML File from the New menu
2. Open the Dojo Controls folder in the Widget palette.
3. Drag the Tree widget onto the canvas.
4. SmartInput is displayed. It should have a preview area on the left and a node properties area on the right. Since no nodes are selected, everything in the properties area should be disabled.

Smart input isn't displayed on chrome or safari.  Instead you get only default content

```
1.
  1.1
  1.2
2.
  2.1
  2.2
```

**Comment by aerwin** [comment]
@peller @childsb Any ideas?

I'm seeing the following errors when I load Maqetta from lab9:50010:

```
Error
arguments: undefined
get stack: function () { [native code] }
info: Object
message: "multipleDefine"
set stack: function () { [native code] }
src: "dojoLoader"
type: undefined
__proto__: SetUpError.d
 dojo.js:1092
GET http://lab9.austin.ibm.com:50010/maqetta/app/metadata/core/0.3/package.json?5688d486c29c4a6accd325722f7ca88edda0c394 404 (ProxyServlet: /maqetta/app/metadata/core/0.3/package.json) dojo.js:4649
xhr dojo.js:4649
dojo.xhr dojo.js:5357
dojo.xhrGet dojo.js:5373
_2114.init davinci.js:45774
_2114.init davinci.js:45771
_935.run davinci.js:11981
run :50010/maqetta/:120
(anonymous function) :50010/maqetta/:139
_c7 dojo.js:603
_36 dojo.js:684
_7a dojo.js:698
_e9 dojo.js:736
_39b dojo.js:4709
_4d8
get stack: function () { [native code] }
message: "Unable to load app/metadata/core/0.3/package.json?5688d486c29c4a6accd325722f7ca88edda0c394 status: 404"
response: Object
set stack: function () { [native code] }
__proto__: TMP
```

And, then when I try to drag tree onto page I get:

```
Error
 dojo.js:1092
_c.uid dojo.js:1092
req.signal dojo.js:157
_9 dojo.js:31
req.signal dojo.js:156
_f5 dojo.js:886
_79 dojo.js:916
_9 dojo.js:31
_79 dojo.js:915
_ef dojo.js:799
req.injectUrl._109 dojo.js:965
GET http://lab9.austin.ibm.com:50010/maqetta/app/metadata/dojo/1.8//HTMLSubElementHelper.js?5688d486c29c4a6accd325722f7ca88edda0c394 404 (ProxyServlet: /maqetta/app/metadata/dojo/1.8//HTMLSubElementHelper.js) dojo.js:975
req.injectUrl dojo.js:975
_32 dojo.js:874
_9 dojo.js:31
_74 dojo.js:345
_79 dojo.js:916
_9 dojo.js:31
_79 dojo.js:915
_ef dojo.js:799
req.injectUrl._109 dojo.js:965
Error
 dojo.js:1092
_c.uid dojo.js:1092
req.signal dojo.js:157
_9 dojo.js:31
req.signal dojo.js:156
(anonymous function) dojo.js:970
```

**Comment by aerwin** [comment]
I'm not sure to what extent the 2nd error (when actually trying to use tree) is influenced by the first error, but the URL used to try and load HTMLSubElement helper doesn't make sense:
- Within my dijit/TreeHelper class I require HTMLSubElement helper using "./HTMLSubElementHelper" so at the very least I'd expect to see "dijit" in the path. 
- Also, there appears to be an extra slash between "1.8" and "HTMLSubElementHelper.js"

**Comment by peller** [comment]
first error, see #2754

**Comment by aerwin** [comment]
Ok, looks like #2754 is fixed (addressing the first issue). It does not appear that the first issue caused the 2nd issue.

The 2nd issue is still quite odd. In my local Eclipse environment, `HTMLSubElementHelper` is successfully loaded with the following URL:

http://localhost:8081/maqetta/app/metadata/dojo/1.8/dijit//HTMLSubElementHelper.js?@revision@

With the build, dijit is missing and an extra "/" appears:

http://lab9:50010/maqetta/app/metadata/dojo/1.8//HTMLSubElementHelper.js

**Comment by aerwin** [comment]
--> Prio from High to Critical

**Comment by aerwin** [comment]
--> @peller 

Moving to @peller as he's agreed to look into the deeper loader issues. Please send back to me if I can help.

**Comment by aerwin** [comment]
--> @aerwin 

Preliminarily, it looks like the changes I'm making for #2796 will fix this one.

**Comment by aerwin** [comment]
Marking closed based on commit 6a1ab9931e7065c1c8554941590c214bd39f1343 put in for #2796.
